ZAPD/HighLevel/HLModelIntermediette.cpp

```diff
@@ -260,21 +260,55 @@ void HLModelIntermediette::FromZDisplayList(HLModelIntermediette* model, ZDispla
 	model->blocks.push_back(lastMat);
 	model->blocks.push_back(mesh);
 	model->blocks.push_back(limb);
-
-	// return model;
 }
 
 void HLModelIntermediette::FromZSkeleton(HLModelIntermediette* model, ZSkeleton* zSkeleton)
 {
 	model->hasSkeleton = true;
 
-	for (size_t i = 0; i < zSkeleton->limbs.size(); i++)
+	// Start at the root skeleton node, go down...
+	ProcessZSkeletonLimb(model, zSkeleton, zSkeleton->limbs[0]);
+
+	/*for (int i = 0; i < zSkeleton->limbs.size(); i++)
 	{
 		// ZLimb* limb = zSkeleton->limbs[i];
 
-		for (size_t j = 0; j < model->blocks.size(); j++)
+		if (limb->dList == nullptr && limb->dListPtr != 0)
+			limb->dList = (ZDisplayList*)zSkeleton->parent->FindResource(limb->dListPtr);
+
+		if (limb->dList != nullptr)
+		{
+			auto cmdTrans = new HLSetTranslation(limb->transX, limb->transY, limb->transZ);
+			cmdTrans->parent = model;
+			model->blocks.push_back(cmdTrans);
+
+			FromZDisplayList(model, limb->dList);
+		}
+
+
+		for (int j = 0; j < model->blocks.size(); j++)
 		{
 		}
+	}*/
+}
```

Comment:
Remove commented code.

Reply:
All the HL stuff is experimental. By it's nature there's going to be commented out stuff while I experiment with what works best.

Reply:
can't you keep the comments on your own local branch though?

Reply:
Fwiw I agree here. To make the code easy to follow for everyone else, it would help if experimental code is on a different branch instead of comments on master, if at all possible